packages/flutter_tools/lib/src/ios/simulators.dart

```diff
@@ -343,7 +344,11 @@ class IOSSimulator extends Device {
 
     // Launch the updated application in the simulator.
     try {
```

Comment:
Please add a small comment linking back to the original issue filled here (since it describes the issue very nicely, and the fix is otherwise not obvious in the code)